lib/pbench/cli/server/user_management.py

```diff
@@ -0,0 +1,205 @@
+import click
+
+from pbench import BadConfig
+from pbench.cli.server import config_setup, pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.server.database.database import Database
+from pbench.server.database.models.users import User
+
+
+# User create CLI
+@click.group("user_group")
+@click.version_option()
+@pass_cli_context
+def cli(ctx):
+    pass  # Entry Point
+
+
+@cli.command()
+@common_options
+@click.option(
+    "--username",
+    prompt=True,
+    required=True,
+    help="pbench server account username (will prompt if unspecified)",
+)
+@click.option(
+    "--password",
+    prompt=True,
+    hide_input=True,
+    required=True,
+    help="pbench server account password (will prompt if unspecified)",
+)
+@click.option(
+    "--email",
+    prompt=True,
+    required=True,
+    help="pbench server account email (will prompt if unspecified)",
+)
+@click.option(
+    "--first-name",
+    prompt=True,
+    required=True,
+    help="pbench server account first name (will prompt if unspecified)",
+)
+@click.option(
+    "--last-name",
+    prompt=True,
+    required=True,
+    help="pbench server account last name (will prompt if unspecified)",
+)
+@pass_cli_context
+def user_create(
+    context: object,
+    username: str,
+    password: str,
+    email: str,
+    first_name: str,
+    last_name: str,
+):
+    try:
+        # Setup the pbench server config and db access
+        config_setup(context, "pbench-user-create")
+
+        user = User(
+            username=username,
+            password=password,
+            first_name=first_name,
+            last_name=last_name,
+            email=email,
+        )
+        user.add()
+        click.echo(f"User {username} registered")
+        rv = 0
+    except Exception as exc:
+        click.echo(exc, err=True)
+        rv = 2 if isinstance(exc, BadConfig) else 1
+
+    click.get_current_context().exit(rv)
+
+
+# User delete CLI
+@cli.command()
+@common_options
+@click.argument("username")
+@pass_cli_context
+def user_delete(context: object, username: str) -> None:
+    context.username = username
```

Comment:
Since you don't call anything which cares about the Click context, I don't think you need to add the username to it -- You didn't do anything similar in `user_create()`, e.g. -- instead, you can just use the `username` parameter variable here.